### PR TITLE
helm-secrets: 3.7.0 -> 3.8.1

### PR DESCRIPTION
**Pull request by yurrriq**
###### Motivation for this change

Update to latest version.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/helm/plugins/helm-secrets.nix [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 stdenv.mkDerivation rec {
   pname = "helm-secrets";
-  version = "3.7.0";
+  version = "3.8.1";
 
   src = fetchFromGitHub {
     owner = "jkroepke";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/helm/plugins/helm-secrets.nix:8]: Missing hash update.

Comment (yurrriq) [reply pkgs/applications/networking/cluster/helm/plugins/helm-secrets.nix:8]: For some reason my machine gave me the same value. I'll try again.

Comment (yurrriq) [reply pkgs/applications/networking/cluster/helm/plugins/helm-secrets.nix:8]: I had to use nix-prefetch-github. Weird.

Comment (jonringer) [reply pkgs/applications/networking/cluster/helm/plugins/helm-secrets.nix:8]: common issue with FOD, hash defines it, not anything else. You need to invalidate it to get a new one

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 128805 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

**Pull request merged**